tool/github/summarize_pr_comments.sh

```
@@ -0,0 +1,40 @@
+          comments(first:50){
+            nodes{
+              path
+              line
+              body
+            }
+          }
+        }
+      }
+    }
+  }
+}'
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)